packages/graalvm/src/main/kotlin/elide/runtime/gvm/internals/intrinsics/js/webstreams/ReadableStreamIntrinsic.kt

```diff
@@ -12,64 +12,12 @@
  */
 package elide.runtime.gvm.internals.intrinsics.js.webstreams
 
-import java.io.InputStream
-import java.io.Reader
-import java.nio.ByteBuffer
 import elide.runtime.gvm.api.Intrinsic
 import elide.runtime.gvm.internals.intrinsics.js.AbstractJsIntrinsic
 import elide.runtime.intrinsics.GuestIntrinsic
-import elide.runtime.intrinsics.js.ReadableStream
 
 /** Implementation of readable streams (via the Web Streams standard). */
 @Intrinsic(global = "ReadableStream") internal class ReadableStreamIntrinsic : AbstractJsIntrinsic() {
-  /**
-   * TBD.
-   */
-  internal class ReadableStreamImpl : ReadableStream {
-
-  }
-
-  /**
-   * TBD.
-   */
-  internal companion object Factory : ReadableStream.Factory<ReadableStreamImpl> {
-    /**
-     * TBD.
-     */
-    override fun empty(): ReadableStreamImpl {
-      TODO("Not yet implemented")
-    }
```

Comment:
Some of these are standardized (e.g. [`from`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/from_static)) and thus would be needed in some host-side object which is used as the class layout and static methods

Reply:
The companion object for `ReadableStream` will implement `ProxyInstantiable` and act as a meta-object, so methods like `from` will be seen as static by guest code.